.github/workflows/release.yml

```diff
@@ -165,6 +165,11 @@ jobs:
           ./scripts/create-app-bundle.sh ${{ matrix.build.target }}
         if: runner.os == 'macOS'
 
+      - name: Create dmg (macOS)
```

Comment:
I see this as part of the bundle step above

Reply:
The dmg requires the binaries to be signed, so I think it's better to keep it like that (`build` -> `sign` -> `create dmg` -> `sign and notarize`). What do you think?

Reply:
If that is a requirement then I don't mind. Do you have a reference somewhere to this requirement?